test/notifications/display_test.dart

```diff
@@ -288,6 +288,66 @@ void main() {
         expectedTagComponent: expectedTagComponent);
     })));
 
+    test('stream message: multiple messages, different topics', () => runWithHttpClient(() => awaitFakeAsync((async) async {
```

Comment:
```
notif test: Add more tests for messaging style notif

Add more tests for Android messaging style notif implementation,
listed here:
    #718 (review)
```

It looks like you meant to link to a specific comment in #718, but there isn't a link to the comment.